docs/scarpet/api/Auxiliary.md

```diff
@@ -252,6 +252,8 @@ Decorators (listed as extra argument after the component they would affect):
  * `'^<format> <text>'` - hover over tooltip text, appearing when hovering with your mouse over the text below.
  * `'?<suggestion>` - command suggestion - a message that will be pasted to chat when text below it is clicked.
  * `'!<message>'` - a chat message that will be executed when the text below it is clicked.
+ * `'@<url>'` - an url will be opened when the text below it is clicked.
+ * `'&<text>'` - a text will be copied to clipboard when the text below it is clicked.
```

Comment:
```suggestion
 * `'@<url>'` - a URL will be opened when the text below it is clicked.
 * `'&<text>'` - some text will be copied to clipboard when the text below it is clicked.
```
Use "a" instead of "an" before "URL" ([Writing Explained](https://writingexplained.org/a-vs-an-difference)).
Capitalize "URL" because it is an acronym.
Use "some text" instead of "a text" as "text" is not necessarily atomic.

Reply:
I'll do the first two, not doing the third one because it's referring to the text given in the parameter, and that's the way it's mentioned in the rest of options (I'll add a `that`, because I forgot it in both).